x-pack/plugins/poc_kibana_cloud_plugin_integration/public/index.scss

```diff
@@ -0,0 +1,2 @@
+// Import the EUI global scope so we can use EUI constants
+@import 'src/legacy/ui/public/styles/_styling_constants';
```

Comment:
I think we can delete this file. If anything, I think it should be moved into the package, since that's where the styles will be consumed. @pugnascotia Cloud uses SCSS, so y'all should be able to consume and build these styles, right?